src/test/scala-2.11/org/apache/james/gatling/jmap/scenari/common/CommonSteps.scala

```diff
@@ -0,0 +1,24 @@
+package org.apache.james.gatling.jmap.scenari.common
+
+import io.gatling.core.Predef._
+import org.apache.james.gatling.control.UserFeeder
+import org.apache.james.gatling.jmap.{JmapAuthentication, JmapMailboxes}
+
+import scala.concurrent.duration._
+
+object CommonSteps {
+
+  def authentication(userCount: Int) =
+    scenario("JmapAuthentication")
+    .feed(UserFeeder.createUserFeederWithInboxAndOutbox(userCount))
+    .pause(1 second, 30 second)
+    .exec(JmapAuthentication.authentication())
+    .pause(1 second)
+
+  def provisionSystemMailboxes(userCount: Int) =
```

Comment:
I think we can definitely say we will always want users provisioned with needed mailbox